lib/dolly/document.rb

```diff
@@ -36,6 +36,10 @@ def update_properties! options = {}
       save
     end
 
+    def reload
+      self.doc = self.class.find(id).doc
```

Comment:
I dont think you need the self in doc... also, please add to the test the check that accessors are updated